README.md

```diff
@@ -93,101 +94,90 @@ Retrieves a list of orders given query parameters. Default is all open orders.
 
 #### Response
 
-[See response schema](https://github.com/0xProject/json-schemas/blob/master/schemas/relayer_api_order_response_schema.ts#L1)
+[See response schema](https://github.com/0xProject/json-schemas/blob/master/schemas/relayer_api_orders_response_schema.ts#L1)
 
 ```
 [
     {
         "signedOrder": {
+            "exchangeContractAddress": "0x12459C951127e0c374FF9105DdA097662A027093",
             "maker": "0x9e56625509c2f60af937f23b7b532600390e8c8b",
             "taker": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
-            "makerFee": "100000000000000",
-            "takerFee": "200000000000000",
-            "makerTokenAmount": "10000000000000000",
-            "takerTokenAmount": "20000000000000000",
             "makerTokenAddress": "0x323b5d4c32345ced77393b3530b1eed0f346429d",
             "takerTokenAddress": "0xef7fff64389b814a946f3e92105513705ca6b990",
-            "salt": "256",
             "feeRecipient": "0xB046140686d052ffF581f63f8136CcE132e857dA",
-            "exchangeContractAddress": "0x12459C951127e0c374FF9105DdA097662A027093",
+            "makerTokenAmount": "10000000000000000",
+            "takerTokenAmount": "20000000000000000",
+            "makerFee": "100000000000000",
+            "takerFee": "200000000000000",
             "expirationUnixTimestampSec": "42",
+            "salt": "256",
```

Comment:
Let's use a more realistic salt here. Take one from the tests.

Reply:
every single salt in schema_test.ts is 256, what would be your suggestion to put here